skorch/tests/callbacks/test_logging.py

```diff
@@ -191,6 +192,139 @@ def test_first_batch_flag(
         npt.on_batch_end(net)
         assert npt.first_batch_ is False
 
+@pytest.mark.skipif(
+    not sacred_installed, reason='Sacred is not installed')
+class TestSacred:
+    @pytest.fixture
+    def net_cls(self):
+        from skorch import NeuralNetClassifier
+        return NeuralNetClassifier
+
+    @pytest.fixture
+    def data(self, classifier_data):
+        X, y = classifier_data
+        # accelerate training since we don't care for the loss
+        X, y = X[:40], y[:40]
+        return X, y
+
+    @pytest.fixture
+    def sacred_logger_cls(self):
+        from skorch.callbacks import SacredLogger
+        return SacredLogger
+
+    @pytest.fixture
+    def sacred_experiment_cls(self):
+        from sacred import Experiment
+        return Experiment
+
+    @pytest.fixture
+    def mock_experiment(self, sacred_experiment_cls):
+        mock = Mock(spec=sacred_experiment_cls)
+        mock.log_scalar = Mock()
+        return mock
+
+    @pytest.fixture
+    def net_fitted(
+            self,
+            net_cls,
+            classifier_module,
+            data,
+            sacred_logger_cls,
+            mock_experiment,
+    ):
+        return net_cls(
+            classifier_module,
+            callbacks=[sacred_logger_cls(mock_experiment)],
+            max_epochs=3,
+        ).fit(*data)
+
+    def test_ignore_keys(
+            self,
+            net_cls,
+            classifier_module,
+            data,
+            sacred_logger_cls,
+            mock_experiment,
+    ):
+        # ignore 'dur' and 'valid_loss', 'unknown' doesn't exist but
+        # this should not cause a problem
+        npt = sacred_logger_cls(
```

Comment:
```suggestion
        logger = sacred_logger_cls(
```
`npt` was for "neptune" ;)

Reply:
Of course! Seems like I took the "inspiration" from the neptune test a little too far ;)